src/clients/locations/FeatureServiceClient.js

```diff
@@ -6,20 +6,28 @@ const trackDependency = require('../appinsights/AppInsightsClient').trackDepende
 
 const apiUrlBase = process.env.FORTIS_FEATURE_SERVICE_HOST;
 
-function formatIdsUri(ids) {
-  return `${apiUrlBase}/features/id/${ids.map(encodeURIComponent).join(',')}?include=bbox`;
+function formatIdsUri(ids, include) {
```

Comment:
Does `include` have to be a parameter passed in to all of these methods? Could it be a member of the `FeatureServiceClient`?